Xamarin.Forms.Core/Button.cs

```diff
@@ -49,9 +49,10 @@ public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonC
 
 		public static readonly BindableProperty ImageProperty = ImageElement.FileImageProperty;
 
-
 		public static readonly BindableProperty PaddingProperty = PaddingElement.PaddingProperty;
 
+		public static readonly BindableProperty MaxLinesProperty = BindableProperty.Create("MaxLines", typeof(int), typeof(Button), 1);
```

Comment:
Please use `nameof(MaxLines)` here instead of string. Also, the default value is better to set -1

Reply:
👍